go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/stretchr/testify v1.8.2
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
-	go.mongodb.org/atlas v0.26.2-0.20230518152935-67df00231ba6
+	go.mongodb.org/atlas v0.25.1-0.20230522162956-d4f376b72f78
```

Comment:
👀 seems like a regression

Reply:
I think we have multiple people using different branches and this keeps going back a forth so it would be good to clarify what branch this is using so people update to the correct one

Reply:
I've made sure this is the newest, I've merged to the sdkv2 branch and updated next